### PR TITLE
Remove Flipper for Dependents Encrypted Jobs (Requires Second PR)

### PR DESCRIPTION
**Pull request by tfink419**
## Summary
- Remove flipper for using encrypted jobs in dependents forms
  - Keep logic for encrypted jobs 
- This goes back to referring to the original job names, however the encrypted jobs still exist as their original code names. This allows a transition for Sidekiq queues in production to all complete gracefully. A second PR will be needed to be merged after this PR has been in production.

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#67515
- department-of-veterans-affairs/vets-api#15116

## Testing done
- Specs were run
- Job was forced to submit through local rails console to see how it handled parameters

## What areas of the site does it impact?
No customer-facing changes as well as no change to logic is expected, this is removing excess flipper logic and associated code

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  ~~Documentation has been updated (link to documentation)~~
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  ~~Feature/bug has a monitor built into Datadog or Grafana (if applicable)~~
- [x]  ~~If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected~~
- [x]  ~~I added a screenshot of the developed feature~~

### CODE REVIEW

**Comment by tfink419** [comment]
Git doesn't accurately represent the LoC truly changed here due to files being renamed and references changed.

The basic logic of changes goes as follows:

`app/sidekiq/bgs/submit_form674_encrypted_job.rb`: Flipper logic removed
`app/sidekiq/bgs/submit_form674_job.rb`: All code deleted, Points to above

`app/sidekiq/bgs/submit_form686c_encrypted_job.rb`: Flipper logic removed
`app/sidekiq/bgs/submit_form686c_job.rb`: All code deleted Points to above

`app/sidekiq/vbms/submit_dependents_pdf_encrypted_job.rb`: Flipper logic removed
`app/sidekiq/vbms/submit_dependents_pdf_job.rb`: All code deleted Points to above

`app/services/bgs/dependent_service.rb` and `spec/services/bgs/dependent_service.rb`: Flipper logic removed

`spec/sidekiq/bgs/submit_form674_job_spec.rb`: Deleted
`spec/sidekiq/bgs/submit_form674_encrypted_job_spec.rb`: Renamed to above

`spec/sidekiq/bgs/submit_dependents_pdf_job.rb`: Deleted
`spec/sidekiq/bgs/submit_form686c_encrypted_job.rb`: Renamed to above

`spec/sidekiq/bgs/submit_dependents_pdf_job.rb`: Deleted
`spec/sidekiq/bgs/submit_dependents_pdf_encrypted_job.rb`: Renamed to above

**Review by tblackwe — APPROVED** [review]

File: config/features.yml [hunk 1]
```diff
@@ -330,10 +330,6 @@ features:
     actor_type: user
     description: Manage dependent removal from view dependent page
     enable_in_development: true
-  dependents_encrypt_jobs:
-    actor_type: user
-    description: Manage whether Sidekiq Jobs used for Dependents Claims will encrypt arguments considered potential PII.
-    enable_in_development: true
```
Comment (rmtolmach) [inline config/features.yml:336]: Once this is deployed, you'll need to follow step 3 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.

**Review by tblackwe — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]

**Comment by rmtolmach** [comment]
@tfink419 or @tblackwe, since i'll need to override the danger check, @ me when you want me to merge this.

**Comment by tblackwe** [comment]
@rmtolmach I am ready to merge this in

**Pull request merged**